frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -198,60 +242,28 @@ export class ArchivedItemDetail extends TailwindElement {
     switch (this.activeTab) {
       case "qa":
         sectionContent = this.renderPanel(
-          html`${this.renderTitle(msg("Quality Assurance (QA)"))}
```

Comment:
I added "(QA)" since we don't otherwise specify in the app what QA abbreviates. If the move is to remove (QA), can we add an info popup somewhere that provides the expansion of the abbreviation? cc @Shrinks99

Reply:
+1 for naming tab "Quality Assurance"